locales/pt_BR.json

```diff
@@ -147,35 +147,35 @@
   "label_keyboard-shortcuts_action_speed-decrease": "",
   "settings_keyboard-shortcuts_help": "Utiliza a sintaxe Mousetrap",
 
-  "text_contributors": "",
+  "text_contributors": "Quem contribuiu: ",
 
-  "label_appearance": "",
-  "label_dark_mode": "",
-  "label_custom_scrollbar": "",
-  "settings_custom_css_help": "",
+  "label_appearance": "Aparência",
+  "label_dark_mode": "Modo noturno",
+  "label_custom_scrollbar": "Personalizar barra de rolagem",
+  "settings_custom_css_help": "Crie um arquivo style.css no diretório de configuração pra costumização personalizada.",
 
   "label_home": "Início",
   "label_home-date-separator-interval": "Agrupar episódios no início por:",
   "label_home-date-separator-interval_value_none": "Desativar",
   "label_home-date-separator-interval_value_day": "Dia",
   "label_home-date-separator-interval_value_month": "Mês",
 
-  "label_download_settings": "",
-  "label_download_directory": "",
-  "button_download_directory_browse": "",
+  "label_download_settings": "Downloads",
+  "label_download_directory": "Pasta de Download",
+  "button_download_directory_browse": "Explorar",
 
-  "label_queue": "",
-  "label_queue_always_remove_finished": "",
-  "label_queue_auto_download": "",
+  "label_queue": "Fila",
+  "label_queue_always_remove_finished": "Sempre remova os episódios após a conclusão",
+  "label_queue_auto_download": "Baixe automaticamente os episódios na fila",
 
-  "label_settings_other": "",
-  "label_minimize_to_taskbar": "",
-  "label_close_to_taskbar": "",
-  "label_smooth_scrolling": "",
+  "label_settings_other": "Outros",
+  "label_minimize_to_taskbar": "Minimizar",
```

Comment:
Would something like "Minimizar para a bandeja" be more suitable in this context, since this is a toggle to enable/disable minimizing to the taskbar?

Reply:
Hey, sure. "Minimizar para a bandeja" makes more scene in that context.